src/metabase/pulse/util.clj

```diff
@@ -42,7 +42,8 @@
                                 (process-query))
                               (process-query))]
           {:card   card
-           :result result}))
+           :result result
+           :type   :card}))
```

Comment:
adding a :type here so `execute-card` this is considered a `part`, which is an internal data type that both alert and dashboard subscription usees for rendering.